maven-plugin/src/main/java/org/openmrs/maven/plugins/Setup.java

```diff
@@ -310,6 +348,39 @@ private void downloadOWAs(File targetDirectory, DistroProperties distroPropertie
 		}
 	}
 
+	private void setConfigFolder(Server server, DistroProperties distroProperties) throws MojoExecutionException {
+		if(distroProperties != null) {
+			File configDir = new File(server.getServerDirectory(), SDKConstants.OPENMRS_SERVER_CONFIGURATION);
+			configDir.mkdir();
+			downloadConfigs(distroProperties, configDir);
+			File configFile = new File(configDir, "referenceapplication-config.owa");
```

Comment:
Do you know why this is named as `referenceapplication-config.owa`?

Reply:
Maven package `org.openmrs.distro:referenceapplication-config:{version}` contains` referenceapplication-config.owa`.